### PR TITLE
MML- 170 - allowed line break in textarea

### PR DESCRIPTION
**Pull request by psanchez-daitangroup**

### CODE REVIEW

File: src/test/java/org/symphonyoss/symphony/messageml/elements/form/TextAreaTest.java [hunk 1]
```diff
@@ -16,13 +16,16 @@
   private static final String NAME_VALUE = "A name";
   private static final String PLACEHOLDER_VALUE = "A placeholder";
   private static final String INITIAL_VALUE = "An initial value";
+  private static final String LINE_BREAK = "An \n initial \n value";
```
Comment (micaste) [inline src/test/java/org/symphonyoss/symphony/messageml/elements/form/TextAreaTest.java]: `LINE_BREAK` could you name this constant another way ? While reading the code, my instinct tells me that this variable equals `\n`. You can maybe name it `INITIAL_VALUE_WITH_LINE_BREAK`

Comment (psanchez-daitangroup) [reply src/test/java/org/symphonyoss/symphony/messageml/elements/form/TextAreaTest.java]: sure

**Comment by psanchez-daitangroup** [comment]
ok to test

File: src/main/java/org/symphonyoss/symphony/messageml/elements/TextNode.java [hunk 2]
```diff
@@ -41,7 +41,7 @@ public TextNode(Element parent, String text) {
   @Override
   public void asPresentationML(XmlPrintStream out) {
     String content = this.text;
-    if (out.isRemoveNl() && !(getParent() instanceof Code)) { // Do not remove newlines if the parent node is a code node
+    if (out.isRemoveNl() && !(getParent() instanceof Code) && !(getParent() instanceof TextArea)) { // Do not remove newlines if the parent node is a code node
```
Comment (lmacedodaitan) [inline src/main/java/org/symphonyoss/symphony/messageml/elements/TextNode.java]: Update this comment to contemplate TextArea as well

Comment (psanchez-daitangroup) [reply src/main/java/org/symphonyoss/symphony/messageml/elements/TextNode.java]: done

File: src/test/java/org/symphonyoss/symphony/messageml/elements/form/TextAreaTest.java (hunk 1 above)
Comment (psanchez-daitangroup) [reply src/test/java/org/symphonyoss/symphony/messageml/elements/form/TextAreaTest.java]: done

**Pull request merged**